includes/SpecialVerifiedExport.php

```diff
@@ -30,6 +32,9 @@ public function __construct( ILoadBalancer $loadBalancer = null ) {
 		$this->loadBalancer = $loadBalancer ?? MediaWikiServices::getInstance()->getDBLoadBalancer();
 	}
 
+	/**
+	 * @param string|null $par
```

Comment:
Is this in the form of doc block because PHP 7.4 doesn't have union operator yet?

Reply:
exactly. php also does not have specified arrays like Title[] or string[] etc.
In this cases we should use a docblock